### DISCUSSION TITLE
Not able to see table in newly added schema

### DISCUSSION THREAD

**Discussion by ravikatariyaunzer** [getredash/redash#6943] [Q&A]
![Screenshot 2024-05-03 at 10 12 09](https://github.com/getredash/redash/assets/151016098/66b9d688-9882-470c-8631-37820bd62544)
I have added new data source but i am not able to see the tables inside it i tried many things but still i am not able to see the tables? could anyone help ?

**Comment by justinclift** [comment]
If you connect to the same data source manually (ie through the standard data source admin tools), using the same authentication details (ie same user/password), does that user see the table?

Asking because it *might* be a permissions problem with the newly created table.  Or it might be in a schema which that user doesn't normally see by default (depends on the data source).

**Reply by ravikatariyaunzer** [reply]
yes user have the permission if then run the query in redash with a table name then it is also working but tables are not visible

**Reply by justinclift** [reply]
Interesting.  Which version of Redash are you using, and what type of data source is it?

**Comment by gaecoli** [comment]
Please show your connection settings.
